server/src/App/Controllers/EventController.php

```diff
@@ -30,13 +30,16 @@ public function getAll(Request $request, Response $response): Response
         $startDate = $request->getQueryParam('start', null);
         $endDate = $request->getQueryParam('end', null);
         $deleted = (bool)$request->getQueryParam('deleted', false);
+        $withMaterials = (bool)$request->getQueryParam('materials', false);
 
         $results = (new Event)
             ->setSearchPeriod($startDate, $endDate)
             ->getAll($deleted)
             ->with('Beneficiaries:persons.id,first_name,last_name')
             ->with('Technicians');
 
+        if($withMaterials == true) $results->with('Materials');
```

Comment:
Le formatage est à revoir.

```suggestion
        if ($withMaterials == true) {
            $results->with('Materials');
        }
```

Reply:
Cf. https://github.com/Robert-2/Robert2/pull/254/checks?check_run_id=3370322887